tests/core/input.cddl

```diff
@@ -185,6 +185,9 @@ inline_wrapper = [{ * text => text }]
 top_level_array = [* uint]
 top_level_single_elem = [uint]
 
+wrapper_table = { * uint => text } ; @newtype
+wrapper_list = [ * uint ] ; @newtype
```

Comment:
I noticed you added tests for these, but didn't actually check anywhere in the rust part of the test if these actually did anything

Reply:
Testing it compiles. The wrapper serialization code is the same as the other code tested elsewhere. I guess I should test it's even generated though.

Reply:
Added some quick ones